.github/CODEOWNERS

```diff
@@ -54,9 +54,9 @@ gradlew.bat @BenHenning
 *Test.kt @anandwana001
 
 # All resource files.
```

Comment:
For this & other lines, can you add TODOs like:

``TODO(#issue): Replace BenHenning with rt4914 after 25-04-2021.``

or

``TODO(#issue): Re-add rt4914 after 25-04-2021.`` (for items where we share ownership normally).

This also requires filing an issue to tracking re-adding you.

This is all convention from Oppia web to help ensure we don't forget re-adding you. :)

Reply:
Done. Thanks.